avl_tree/AvlTree.java

```diff
@@ -0,0 +1,306 @@
+import java.util.Scanner;
+import java.util.NoSuchElementException;
+import java.util.InputMismatchException;
+
+public class AvlTree {
+
+	public static void main(String[] args) {
+		AVL avl = new AVL();
+		Scanner inp = new Scanner(System.in);
+		while (true) {

```

Comment:
I stated that earlier, There is no sense of making boolean variable assign it true and use it as argument because eventually i had to ```break``` the loop.